backend/models/dynamodb/Entity.js

```diff
@@ -0,0 +1,87 @@
+
+import DynamoDB from "@klaudsol/commons/lib/DynamoDB"
+import { 
+  DYNAMO_DB_TYPES, 
+  DYNAMO_DB_INDEXES, 
+  DYNAMO_DB_TABLE, 
+  DYNAMO_DB_ORDER 
+} from "@/constants";
+
+export default class Entity {
+  // Gets all items where entity = entity_type_slug (eg. tech)
+  // and if type = content
+  // Sample: api/tech
+  // TODO: Add pagination
+  // TODO: Add filtering
+  // TODO: Add search
+  static async whereEntityTypeSlug({ entity_type_slug, order }) {
+    const db = new DynamoDB();
+
+    let scanIndexForward = true;
+    if (order === DYNAMO_DB_ORDER.desc) {
+      scanIndexForward = false;
+    } 
```

Comment:
I think it makes more sense to default to descending, so that by default newer (and probably more relevant) entries are shown first.
```suggestion
    let scanIndexForward = false;
    if (order === DYNAMO_DB_ORDER.asc) {
      scanIndexForward = true;
    } 
```